<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Puzzle8_Solver</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/prefix1/Main_icon.png</normaloff>:/new/prefix1/Main_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbl_Back">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>11</y>
      <width>576</width>
      <height>525</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 4px solid rgb(204, 0, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_input">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>10</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Input</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_solve">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>50</y>
      <width>215</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&amp;Solve</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_refresh">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_algo">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>100</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1.5px solid blue;
background-color: rgb(5, 203, 169);

</string>
    </property>
    <property name="text">
     <string>Algorithm :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_algorithm">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>UCS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BFS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DFS</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="txt_solution">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>180</y>
      <width>215</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_solution">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>155</y>
      <width>215</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 2px solid rgb(204, 0, 0);
background-color: rgb(245, 133, 133);</string>
    </property>
    <property name="text">
     <string>Solution : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_r1">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>360</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 2px solid rgb(13, 10, 161);
background-color: rgb(173, 127, 168);</string>
    </property>
    <property name="text">
     <string>Moves : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_moves">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>360</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 2px solid rgb(13, 10, 161);
background-color: rgb(173, 127, 168);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_r2">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>405</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 2px solid rgb(9, 93, 12);
background-color: rgb(138, 226, 52);</string>
    </property>
    <property name="text">
     <string>Time(ms) :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_time">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>405</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 2px solid rgb(9, 93, 12);
background-color: rgb(138, 226, 52);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_nextstep">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>450</y>
      <width>215</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Next Step</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_info">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>500</y>
      <width>215</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 2px solid rgb(251, 73, 3);
background-color: rgb(237, 212, 0);</string>
    </property>
    <property name="text">
     <string>Poorya mohammadi nasab</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_a1">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>26</y>
      <width>172</width>
      <height>155</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid blue;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/imgs/1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_a2">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>26</y>
      <width>172</width>
      <height>155</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid blue;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/imgs/2.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_a3">
    <property name="geometry">
     <rect>
      <x>398</x>
      <y>26</y>
      <width>172</width>
      <height>155</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid blue;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/imgs/3.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_a4">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>196</y>
      <width>172</width>
      <height>155</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid blue;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/imgs/4.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_a5">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>196</y>
      <width>172</width>
      <height>155</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid blue;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/imgs/5.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_a6">
    <property name="geometry">
     <rect>
      <x>398</x>
      <y>196</y>
      <width>172</width>
      <height>155</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid blue;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/imgs/6.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_a7">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>366</y>
      <width>172</width>
      <height>155</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid blue;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/imgs/7.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_a8">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>366</y>
      <width>172</width>
      <height>155</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid blue;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/imgs/8.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Blank">
    <property name="geometry">
     <rect>
      <x>398</x>
      <y>366</y>
      <width>172</width>
      <height>155</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid blue;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/imgs/Blank.JPG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
